module/localStorage.js

```diff
@@ -1,44 +1,44 @@
 // eslint-disable-next-line
-import Booklibrary from './books.js';
-
 export default class Books {
-    booklibrary = new Booklibrary();
 
-  static books = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+  static getfromLS = () => (localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [])
 
-  static add(author, title) {
-    const book = { title, author };
-    Books.books.push(book);
-    localStorage.setItem('books', JSON.stringify(Books.books));
-    Books.display();
+  static idGenerator = () => {
+    const books = Books.getfromLS();
+    const id = books.length ? books[books.length - 1].id + 1 : 1;
+    return id;
   }
 
-  static remove(author, title) {
-    for (let i = 0; i < Books.books.length; i += 1) {
-      if (Books.books[i].title === title && Books.books[i].author === author) {
-        Books.books.splice(i, 1);
-      }
-    }
-    //   localStorage.clear();
-    localStorage.setItem('books', JSON.stringify(Books.books));
-    Books.display();
+  static add = (book) => {
+    const books = Books.getfromLS();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+
+    Books.display(book);
   }
```

Comment:
**[Optional]**
- [ ] After adding a book your page should not reload because you have made it your Single Page Application that is not expected to reload. Please fix this issue :+1: